packages/thumbprint-react/components/DatePicker/index.module.scss

```diff
@@ -47,11 +35,11 @@
 }
 
 .root :global(.DayPicker-NavButton) {
-    top: 130px;
-    border: 1px solid $tp-color__gray;
+    top: 0;
+    border: 0;
     position: absolute;
-    width: 40px;
-    height: 90px;
+    width: 75px;
+    height: 1.5rem;
```

Comment:
What's the benefit of using `rem` for the `height` but pixels for the `width`? Would the arrow get stretched vertically?

Reply:
I made some more improvements:
- I converted few things into px because the text size is in px. The vertical alignment with the month button breaks when the units are inconsistent (the button stretches but the text doesn't).
- Increased the touch target of the month button to 44px.